test/test_feature_library.py

```diff
@@ -224,11 +224,9 @@ def test_sindypi_library_bad_params(params):
 @pytest.mark.parametrize(
     "library",
     [
-        IdentityLibrary(),
         PolynomialLibrary(),
         PolynomialLibrary(include_bias=False),
         FourierLibrary(),
-        IdentityLibrary() + PolynomialLibrary(),
```

Comment:
We still want to test that the sum of two libraries works as expected.  Keep this line.